ts/Dashboards/Layout/Cell.ts

```diff
@@ -584,9 +584,6 @@ class Cell extends GUIElement {
 
 namespace Cell {
 
-    /**
-     * @internal
-     **/
     export interface CellResponsiveOptions {
         width: (string|number);
```

Comment:
Width needs to be described, can `visible` be removed?

Reply:
Sure :D